browser/src/control/Control.Header.ts

```diff
@@ -548,6 +548,7 @@ export class Header extends app.definitions.canvasSectionObject {
 			this._prevMouseOverEntry = this._mouseOverEntry;
 			this._mouseOverEntry = result.entry;
 		}
+		else return;
```

Comment:
what about dragging mode (the next `else`), does it still work correctly?

Reply:
?

Reply:
It works correctly yes. I remember answering this, maybe i wrote it at some wrong place.